BossMod/Components/Knockback.cs

```diff
@@ -30,6 +30,7 @@ protected struct PlayerImmuneState
         {
             public DateTime ArmsLengthSurecastExpire; // 0 if not active
             public DateTime InnerStrengthExpire; // 0 if not active
+            public DateTime ManawallExpire; // 0 if not active
 
             public bool ImmuneAt(DateTime time) => ArmsLengthSurecastExpire > time || InnerStrengthExpire > time;
```

Comment:
You also need to include the check here for it to actually work.

Reply:
fixed it